CHANGELOG.md

```diff
@@ -12,7 +12,7 @@
     - `simple_string`: Provides a simple string representation of the data.
     - `json_value`: Returns the data as a JSON-compatible value.
     - `json`: Converts the data to a JSON string.
-  - To `ArrayType`, `MapType`, `StructField`, and `StructType`:
+  - To `ArrayType`, `MapType`, `StructField`, `PandasSeriesType`, `PandasDataFrameType` and `StructType`:
```

Comment:
Should vector be there as well?

Reply:
No. VectorType is an atomic type and does not have a from_json() function

Reply:
Ah, I see this note is for from_json. Should there be a note for vectors `json_value` function?